### PR TITLE
Share on Google+ is now aligned

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #[1833](https://github.com/fossasia/chat.susi.ai/issues/1833#issue-390240446)

Changes:  Share on Google+ is now aligned

Screenshots for the change: 

Before:

social

After:

social1

### CODE REVIEW

**Comment by AakashMallik** [comment]
@divyabaid16 Please provide a deployment link with your PR.

File: src/components/ChatApp/Settings/ShareOnSocialMedia.js [hunk 1]
```diff
@@ -96,14 +96,14 @@ const ShareOnSocialMedia = props => {
...
-            <Translate text="Share about SUSI on Google +" />
+            <Translate text="Share about SUSI on Google+" />
             <br />
             <RaisedButton
               label={<Translate text="Share on Google+" />}
               style={raisedButtonStyle}
               backgroundColor="#d34836"
               labelColor="#fff"
-              icon={<FontIcon className="fa fa-google-plus" />}
+              icon={<FontIcon className="fa fa-google-plus-circle" />}
```
Comment (AakashMallik) [inline src/components/ChatApp/Settings/ShareOnSocialMedia.js]: This icon is inconsistent with the other icons. Please revert it back to **fa-google-plus**

**Comment by divyabaid16** [comment]
@AakashMallik, I think that this issue is browser specific. It was working fine with the Mozilla Developers browser but it had some problem with Mozilla firefox. And also, all other font icon are working fine except for the google plus.

**Comment by AakashMallik** [comment]
@divyabaid16 So that white space was cause by the google plus icon?

**Comment by divyabaid16** [comment]
@AakashMallik , here the problem lies.
selection_038

I guess all the three are not consistent with each other.

**Comment by AakashMallik** [comment]
@divyabaid16 So the ideal fix would be to fix the button without changing the icons being used inside them. Also, let us try using Material-ui icons from the material-ui library instead of the current implementation. That would also make the code more consistent to the rest of the code base.

**Comment by divyabaid16** [comment]
@AakashMallik  I have done the changes. The Material-UI icon was not available for these shares icons so I haven't changed them.

**Comment by AakashMallik** [comment]
@divyabaid16 

> @AakashMallik I have done the changes. The Material-UI icon was not available for these shares icons so I haven't changed them.

Yeash my bad, got confused with this repo https://react-icons.netlify.com/#/ . Good job btw. Will drop a review soon. :+1:

**Comment by divyabaid16** [comment]
> @AakashMallik, I think that this issue is browser specific. It was working fine with the Mozilla Developers browser but it had some problem with Mozilla firefox. 

@isuruAb This issue is present in my Mozilla Firefox only.

**Pull request merged**